sklearn/manifold/t_sne.py

```diff
@@ -70,10 +73,11 @@ def _joint_probabilities_nn(distances, neighbors, desired_perplexity, verbose):
 
     Parameters
     ----------
-    distances : array, shape (n_samples * (n_samples-1) / 2,)
-        Distances of samples are stored as condensed matrices, i.e.
-        we omit the diagonal and duplicate entries and store everything
-        in a one-dimensional array.
+    distances : array, shape (n_samples, K)
+        Distances of samples to its K nearest neighbors.
+
+    neighbors : array, shape (n_samples, K)
+        K nearest-neighbors for each samples.
```

Comment:
What does it contains? The indices of the neighbors? It might be useful to indicate it.